tests/removeController.test.ts

```diff
@@ -14,35 +14,31 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import Arweave from 'arweave';
-import { JWKInterface, Warp } from 'warp-contracts';
-
 import { ANTState } from '../src/types';
-import { ANTDeployer } from '../tools/common/helpers';
+import { arweave, getLocalWallet, warp } from './utils/helper';
 
-describe('Testing removeController...', () => {
-  const arweave: Arweave = global.arweave;
-  const wallets: Record<string, JWKInterface> = global.wallets;
-  const warp: Warp = global.warp;
-  const defaultOwner = Object.entries(wallets)[0];
+describe('removeController', () => {
+  let antContractTxId: string;
+  let antContractOwnerAddress: string;
+  let contract;
 
-  it('Should remove controller from the ANT', async () => {
-    const ANT = await ANTDeployer(warp, {
-      address: defaultOwner[0],
-      wallet: defaultOwner[1],
-    });
```

Comment:
deploying an ANT for every test increases total testing time, so we now only deploy once